plugins/learneth/profile.json

```diff
@@ -0,0 +1,12 @@
+{
+ "name": "LearnEth",
+ "displayName": "LearnEth",
+ "description": "Learn Ethereum with Remix !",
+ "documentation": "https://github.com/bunsenstraat/remix-learneth-plugin",
+ "version": "0.1.0-beta.3",
+ "methods": [],
+ "kind": "none",
+ "icon": "https://image.flaticon.com/icons/svg/1570/1570493.svg",
```

Comment:
I think that this one is a better icon: https://fontawesome.com/icons/book-reader?style=solid
Changing to this icon involves either finding a cdn with the svg on it or downloading the svg and changing it to a base64 image and putting that in the profile.  
But let me know what you think of the book-reader icon.  I think it fits with the other icons more than the icon from flaticon.

Reply:
https://upload.wikimedia.org/wikipedia/commons/d/da/Font_Awesome_5_solid_book-reader.svg

Reply:
Don't use the base64 - but put the image in a folder in the remix-workshops repo and then put a link to the file in the profile.

Reply:
I can't link the icon in the repo. If you test it it won't be there unless it's in master. So it put it in IPFS. Hope that works.

Reply:
We can't load it from IPFS.  Put it in the repo and I'll test it manually ( with the browser tools).